fdb-record-layer-lucene/fdb-record-layer-lucene.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
     testRuntimeOnly "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
     testRuntimeOnly "org.apache.logging.log4j:log4j-core:${log4jVersion}"
 
+    testImplementation("com.sun.mail:javax.mail:${javaxMail}")
```

Comment:
Is the `javaxMail` property defined? I'd also suggest `javaxMailVersion`, again for consistency with the other dependency version properties (and to put the definition in `gradle.properties`)